src/test/java/gherkin/GherkinDialectProviderTest.java

```diff
@@ -2,13 +2,37 @@
 
 import org.junit.Test;
 
+import java.util.List;
+
 import static gherkin.SymbolCounter.countSymbols;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 public class GherkinDialectProviderTest {
     @Test
     public void providesEmojiDialect() {
         GherkinDialect em = new GherkinDialectProvider().getDialect("em", null);
         assertEquals(1, countSymbols(em.getScenarioKeywords().get(0)));
     }
+
+    @Test
+    public void should_include_swedish_in_languages() {
+        GherkinDialectProvider gherkinDialectProvider = new GherkinDialectProvider();
+        List<String> actual = gherkinDialectProvider.getLanguages();
+
+        String expected = getExpected(gherkinDialectProvider);
+
+        assertTrue("Swedish with explanation should be available", actual.contains(expected));
```

Comment:
I think that `assertThat("Swedish with explanation should be available", actual, CoreMatchers.containsString(expected))` is a bit more readable and provides better diagnostics when the string doesn't contain the expected value. 

Unfortunately it looks like we're using `mockito-all` in this project which bundles a version of `hamcrest-core` that doesn't have `containsString`. 

I'll add a commit to resolve this